modules/ROOT/pages/to-find-info.adoc

```diff
@@ -19,14 +19,20 @@ the HTTP functions in the REST specification.
 
 == Exchange Search
 
-Search lets you find Exchange assets that contain one or more terms
-in an asset title, asset ID, or a tag in an asset. You can also combine search terms with the filters listed in All Types.
-Searches only apply to the Exchange you are viewing.
-If you search in a private Exchange, the search only lists the
-private Exchange assets and not public Exchange assets.
+Exchange Search looks for terms in asset names, tags, categories, properties defined in API specifications, custom attributes (also known as custom fields), and the portal content shown on the asset page. Search includes RAML and OAS APIs and all other assets in Exchange.
+
+Assets migrated from Exchange 1 also allow the search to include text in an asset's description.
```

Comment:
@LucasLudueno Is this still valid☝️?

Reply:
Seconding this question. The current doc at https://docs.mulesoft.com/exchange/to-find-info says "Assets migrated from Exchange 1 also allow the search to include text in an asset’s description." Is this correct?

Reply:
This last note for Exchange 1 assets is not valid anymore since those contents are also in the portal and that's what's being indexed now

Reply:
Thank you! I removed this sentence.